build/release.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -xe
```

Comment:
Might make sense to change the return instead of exit for lines https://github.com/box/box-annotations/pull/106/files#diff-fff440587f66b0e9a460aa8cf04ce6d2L212 and https://github.com/box/box-annotations/pull/106/files#diff-fff440587f66b0e9a460aa8cf04ce6d2L220

Reply:
Then the updated readme/changelog will be reset to it's state before the release was cut